module/Activity/Module.php

```diff
@@ -52,12 +54,27 @@ public function getServiceConfig()
                 'activity_doctrine_em' => function ($sm) {
                     return $sm->get('doctrine.entitymanager.orm_default');
                 },
+                'activity_form_activity' => function ($sm) {
+                    /** @var \Decision\Service\Member $userService */
+                    $memberService = $sm->get('decision_service_member');
+                    /** @var \User\Model\User $identity */
+                    $identity = $sm->get('user_role');
+
+                    // This is a necessary hack, since the user and corresponding member model is cached
+                    // I could not get the organ from the Member, because this resulted in all kinds of exceptions
+                    // This forces doctrine to fetch a clean object from the database
+                    $user = $memberService->findMemberByLidNr($identity->getLidnr());
+
```

Comment:
The most simplest thing would be if I just could use the object methods to get to the needed information, as is (a) expected, and (b) is the way to do it in OOP and (c) minimizes dependencies.

There is already a method in the member service to get the organs a member is in I see now. Since I get the feeling that fixing the behaviour is a lot of work, I will use this method instead.